dist/jest.config.js

```diff
@@ -0,0 +1,5 @@
+export const preset = "ts-jest";
```

Comment:
what is dist folder? should we commit?

Reply:
In the dist folder is the result of the TypeScript compilation process, where TypeScript code is converted into JavaScript code.
I didn't know what best practices are, researched the internet and found out that better to keep it locally. I will add it to the .gitignore folder.